packages/itwin/ec3-widget/package.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Keywords are missing. Please grab them from the OneClickLCA widget.

Reply:
Keywords have been added